community/index.html

```diff
@@ -152,6 +170,24 @@ <h1 class="h3">
         </aside>
     </div>
     <div class="cb person-bio">
+        <article>
+            <figure class="grid-3 l">
+                <img class="bio-img sw" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/default-sw/julia-simplicio.jpg" width="152" height="152" alt="Portrait photo(black and white) of Julia Simplicio." />
+                <img class="bio-img bunt" src="http://hoodiehq.github.io/hoodie-css/src/content_img/community/hover-bunt/julia-simplicio--hover.jpg" width="152" height="152" alt="Portrait photo(colour) of Julia Simplicio." />
+            </figure>
+            <span class="h6">UX Developer / Designer</span>
+            <h1 class="h3">
+              <a name="julia-schmidt" class="jump-mark">Julia Simplicio</a>
+            </h1>
+            <p>
+                Design-focused developer helping to bring more designers to Open Source.
+            </p>
+        </article>
+        <aside class="bio-link">
+            <a href="https://github.com/jsimplicio" target="_blank">Twitter</a> | <a href="https://github.com/jsimplicio" target="_blank">GitHub</a>
```

Comment:
Not sure Julia has a twitter, maybe we could link to their website http://juliasimplicio.com/